src/Popper.js

```diff
@@ -64,13 +67,24 @@ export class InnerPopper extends React.Component<PopperProps, PopperState> {
 
   state = {
     data: undefined,
+    placement: undefined,
+    lastPropsPlacement: undefined,
   };
 
   popperInstance: ?PopperJS$Instance;
 
   popperNode: ?HTMLElement = null;
   arrowNode: ?HTMLElement = null;
 
+  static getDerivedStateFromProps(nextProps: PopperProps, state: PopperState) {
+    const { placement } = nextProps;
+
+    if (nextProps.placement !== state.lastPropsPlacement)
```

Comment:
`if (placement !==` since you just made the variable.

could even deconstruct in the signature itself.
```
static getDerivedStateFromProps({ placement }: PopperProps, { lastPropsPlacement }: PopperState)
```